helm-deployment/charts/scrap-me-subchart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Aren't those Chart.yaml files auto generated?

Reply:
nope
and these files are required